src/main/java/com/ncc/aif/AidaAnnotationOntology.java

```diff
@@ -63,6 +63,20 @@
             ResourceFactory.createProperty(NAMESPACE + "jsonContent");
     public static final Property INFORMATIVE_JUSTIFICATION =
             ResourceFactory.createProperty(NAMESPACE + "informativeJustification");
+    public static final Property LDC_TIME_PROPERTY =
+            ResourceFactory.createProperty(NAMESPACE + "ldcTime");
+    public static final Property LDC_TIME_START =
+            ResourceFactory.createProperty(NAMESPACE + "start");
+    public static final Property LDC_TIME_END =
+            ResourceFactory.createProperty(NAMESPACE + "end");
+    public static final Property LDC_TIME_TYPE =
+            ResourceFactory.createProperty(NAMESPACE + "timeType");
+    public static final Property LDC_TIME_YEAR =
+            ResourceFactory.createProperty(NAMESPACE + "year");
+    public static final Property LDC_TIME_MONTH =
+            ResourceFactory.createProperty(NAMESPACE + "month");
+    public static final Property LDC_TIME_DATE =
```

Comment:
If you agree with the comments above, update `LDC_TIME_DATE` to `LDC_TIME_DAY` and update the property to "day".